os/fs/smartfs/README.txt

```diff
@@ -1,17 +1,20 @@
-SMARTFS README
-^^^^^^^^^^^^^^
+jSmartFs README
+^^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
-Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
+Journaling - Sector Mapped Allocation for Really Tiny FLASH file system, jSmartFs.
+Journaling is an basic assurance against system crashes and unexpected shutdowns.
```

Comment:
I think Journaling is not an "assurance against system crashes etc...".
Instead, "Journaling tries to provide an assurance of file system integrity during system crash, etc.. "